tests/document/instance.py

```diff
@@ -1552,9 +1593,9 @@ class ParentModel(Document):
                 EmbeddedChildModel)
 
         emb = EmbeddedChildModel(id={'1': [1]})
-        ParentModel(children=emb)._get_changed_fields()
+        ParentModel(child=emb)._get_changed_fields()
```

Comment:
Tests that have zero asserts make me a bit anxious... Could we at least assert that `_get_changed_fields()` returns the right value?

Reply:
makes sense, changing it now